src/Identity/EntityFrameworkCore/test/EF.Test/Utilities/ScratchDatabaseFixture.cs

```diff
@@ -23,9 +22,8 @@ public void Dispose()
         {
             if (_testStore.HasValue)
```

Comment:
```suggestion
            if (_testStore.IsValueCreated)
```

Reply:
Thanks. (I guess we didn't copy the API of `Lazy` verbatim like I remembered! :-))

Reply:
Copied `Nullable<T>` at a guess